public/data/shridharkapshikar.json

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
You are missing an opening { on line 1

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Hi Team, I have corrected the missing { on line 1. Request you to review & do the needful.